packages/mgt-components/src/components/mgt-organization/mgt-organization.scss

```diff
@@ -10,6 +10,10 @@
 @import '../../styles/shared-sass-variables';
 @import './mgt-organization.theme';
 
+$coworker-person-avatar-size: var(--coworker-person-avatar-size, 40px);
```

Comment:
We should probably add these css properties to the JSDoc comments for the person-card so that they get surfaced in the storybook docs page for the person-card